src/test/run-make-fulldeps/cross-lang-lto-clang/Makefile

```diff
@@ -0,0 +1,35 @@
+# Copyright 2019 The Rust Project Developers. See the COPYRIGHT
+# file at the top-level directory of this distribution and at
+# http://rust-lang.org/COPYRIGHT.
+#
+# Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+# http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+# <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+# option. This file may not be copied, modified, or distributed
+# except according to those terms.
+
```

Comment:
I thought that license header is removed from Rust project.

Reply:
I don't know, are they?

Reply:
Maybe I missunderstand this PR #57108.

Reply:
Looks like your're right. I'll remove the headers.